src/main/resources/org/apache/maven/plugins/artifact/buildinfo/not-reproducible-plugins.properties

```diff
@@ -29,8 +29,9 @@ maven-ejb-plugin=3.1.0
 # https://issues.apache.org/jira/browse/MEJB-128
 maven-plugin-plugin=3.5.1
 # https://issues.apache.org/jira/browse/MPLUGIN-326
-maven-remote-resources-plugin=1.7.0
+maven-remote-resources-plugin=3.3.0
 # https://issues.apache.org/jira/browse/MRRESOURCES-114
+# https://issues.apache.org/jira/browse/MRRESOURCES-150
```

Comment:
MRRESOURCES-150 - was resolved in 3.3.0 ... so should it be on not-reproducible build list?